homeassistant/components/unifi/__init__.py

```diff
@@ -1,13 +1,32 @@
 """Support for devices connected to UniFi POE."""
+import voluptuous as vol
+
 from homeassistant.const import CONF_HOST
 from homeassistant.helpers.device_registry import CONNECTION_NETWORK_MAC
 
-from .const import CONF_CONTROLLER, CONF_SITE_ID, CONTROLLER_ID, DOMAIN
+import homeassistant.helpers.config_validation as cv
+
+from .const import (
+    CONF_CONTROLLER, CONF_DETECTION_TIME, CONF_SITE_ID, CONF_SSID_FILTER,
+    CONTROLLER_ID, DOMAIN, UNIFI_CONFIG)
 from .controller import UniFiController
 
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Optional(CONF_DETECTION_TIME): vol.All(
```

Comment:
How will this work if there are multiple accounts configured?

Reply:
it will be global until config entry options can replace it

Reply:
ok

Reply:
Changed to per host + site configuration which equals one config entry